src/pages/campaignWizardPage.ts

```diff
@@ -0,0 +1,24 @@
+import { Page } from '@playwright/test';
+import PlaywrightWrapper from '../helper/wrapper/playwrightWrappers';
+
+export default class CampaignWizardPage {
+  private base: PlaywrightWrapper;
+
+  constructor(private page: Page) {
+    this.base = new PlaywrightWrapper(page);
+  }
+
+  directAdvertiserCompanyType = this.page.locator("//label/*[.=' Direct Advertiser ']");
+  companyNameInput = this.page.locator("//input[@name='select_advertiser']");
+  companyName = this.page.locator("//input[@name='select_advertiser']/..");
```

Comment:
What's the difference between `companyNameInput` and `companyName`? I am asking because `companyName` is confusing me.